lib/store/memory.js

```diff
@@ -244,7 +247,27 @@ module.exports.MatrixInMemoryStore.prototype = {
             this.localStorage.setItem("mxjssdk_memory_filter_" + filterName, filterId);
         }
         catch (e) {}
-    }
+    },
+
+    /**
+     * Store user-scoped account data events
+     * @param {Array<MatrixEvent>} events The events to store.
+     */
+    storeAccountDataEvents: function(events) {
+        var self = this;
+        events.forEach(function(event) {
+            self.accountData[event.getType()] = event;
```

Comment:
I find this is somewhat surprising that `storeAccountDataEvents(Event[])` doesn't actually store all the events in the array if there are >1 events with the same `type`, but instead clobbers with the latest one. I assume this is intentional behaviour, so we should mention the clobbering semantics in the functions docs.